test/jsdom/index.js

```diff
@@ -1638,5 +1638,32 @@ exports.tests = {
     window.location.reload();
     t.equal("errors" in global, false, "there should be no errors global after the call");
     t.done();
+  },
+
+  'document.currentScript is undefined when not executing <script>': function(test) {
```

Comment:
Might be best to move these tests to a new test/living-html/current-script.js and follow the style guidelines therein